### PR TITLE
Add initial files to get started

### PR DESCRIPTION
**Pull request by KasonBraley**

### CODE REVIEW

**Comment by KasonBraley** [comment]
Wait for PR #2 to be merged into development before merging this PR.

**Review by Beers15 — APPROVED** [review]

**Review by cburk2019 — COMMENTED** [review]
Looks good! Let's begin.

**Review by cburk2019 — APPROVED** [review]
Let's do this.

**Pull request merged**
